settings.gradle

```diff
@@ -226,9 +226,16 @@ include ':instrumentation:rxjava-2.0:testing'
 include ':instrumentation:rxjava-2.0:javaagent'
 include ':instrumentation:scala-executors:javaagent'
 include ':instrumentation:servlet:glassfish-testing'
+include ':instrumentation:servlet:servlet-common:library'
 include ':instrumentation:servlet:servlet-common:javaagent'
+include ':instrumentation:servlet:servlet-javax-common:library'
+include ':instrumentation:servlet:servlet-javax-common:javaagent'
+include ':instrumentation:servlet:servlet-2.2:library'
 include ':instrumentation:servlet:servlet-2.2:javaagent'
+include ':instrumentation:servlet:servlet-3.0:library'
 include ':instrumentation:servlet:servlet-3.0:javaagent'
+include ':instrumentation:servlet:servlet-jakarta:library'
+include ':instrumentation:servlet:servlet-jakarta:javaagent'
```

Comment:
do we need the library modules currently? we don't have a library instrumentation story for users for these modules yet

related to my comment here: https://github.com/open-telemetry/opentelemetry-java-instrumentation/pull/2599#pullrequestreview-617046209

Reply:
Since the library part was present previously as `instrumentation-core:servlet-2.2` and I did not know of any plans to remove that separation, I made it consistent across all servlet modules for now.

Reply:
that makes sense to me, we'll probably want these library modules sooner than later anyways

cc: @iNikem @anuraaga in case any opinion here

Reply:
I would generally recommend classifying as library is not using the Javaagent API. Javaagent often depend on library and it's normal, but library should never depend on Javaagent. If we ever create library instrumentation in the future we would need to rename the common artifact then, something we don't want to do if ever marking agent artifacts as non-alpha.

Reply:
If we have reusable code which does not use javaagent API, I think it is a good idea to put that into `library` module.